aspnetcore/includes/signalr-version-notice.md

```diff
@@ -0,0 +1,2 @@
+> [!NOTE]
+> This document details the latest version of ASP.NET Core SignalR. See the [SignalR 1.x documentation](../../aspnet/signalr/) for the ASP.NET Framework version.
```

Comment:
Don't use relative links. I think the (aspnet/aspnet/signalr) is what you want. @guardrex  is that right?

Reply:
Yes ... that will land here :point_right: https://docs.microsoft.com/en-us/aspnet/signalr/